README.md

```diff
@@ -1,83 +1,7 @@
-Takeaway Challenge
-==================
-```
-                            _________
-              r==           |       |
-           _  //            |  M.A. |   ))))
-          |_)//(''''':      |       |
-            //  \_____:_____.-------D     )))))
-           //   | ===  |   /        \
-       .:'//.   \ \=|   \ /  .:'':./    )))))
-      :' // ':   \ \ ''..'--:'-.. ':
-      '. '' .'    \:.....:--'.-'' .'
-       ':..:'                ':..:'
+Takeaway Challenge redone in Javascript. 
 
- ```
+Code is unfinished as I have not fully implemented the front end pages, I have yet to persist user log-in, and I still need to store completed orders in the orders DB. 
 
-Instructions
--------
+If you want to take a look at the site, run 'node server.js' in the terminal after installing all the dependencies in the package.json file. 
```

Comment:
When I follow these instructions, I get `error: role "emmanuel" does not exist` and I can't run the page